sass/styleguide.scss

```diff
@@ -0,0 +1,117 @@
+// @file Styleguide Styles
+
+	@import "styles";
+
+// @todo: sassify my color variables into a loop
+// @todo: bourbonize the input fields
+
+/**
+ * Page Globals
+ */
+
+.styleguide-section {
+	@include clearfix();
+	margin-bottom: 2em;
+	&-title {
```

Comment:
This syntax breaks older SASS. Might want to note this somewhere in the readme what the minimum SASS version is because of this usage.